spec/signable/query/response_spec.rb

```diff
@@ -1,34 +1,40 @@
-require "spec_helper"
+# frozen_string_literal: true
 
-describe Signable::Query::Response do
+require 'spec_helper'
 
-  describe "#ok?" do
-    let(:response) { Signable::Query::Response.new http_response }
+# rubocop:disable RSpec/VerifiedDoubles
+describe Signable::Query::Response do
+  describe '#ok?' do
     subject { response.ok? }
 
-    context "when code is 200" do
-      let(:http_response) { double 'http_response', code: 200 }
-      it { should be true }
+    let(:response) { described_class.new http_response }
+
+    context 'when code is 200' do
+      let(:http_response) { double code: 200 }
+
+      it { is_expected.to be true }
```

Comment:
```suggestion
      it { is_expected.to eq true }
```
For this reason: https://github.com/smartpension/api/pull/12217#discussion_r421541928

Reply:
f139680